api/samples/cbr.c

```diff
@@ -232,10 +232,9 @@ insert(hash_table_t table, app_pc addr, cbr_state_t state)
 static void
 at_taken(app_pc src, app_pc targ)
 {
-    dr_mcontext_t mcontext = {
-        sizeof(mcontext),
-        DR_MC_ALL,
-    };
+    dr_mcontext_t mcontext;
```

Comment:
commit message style: No "IX: " in the title (just on the branch name) -- so the title would just start with "Opt..."

Reply:
Thanks. Done.